lib/git_hooks/pre_commit/rspec.rb

```diff
@@ -18,6 +18,10 @@ def validate
 
         abort 'Prevented broken commit' if rspec_executor.errors?
       end
+
+      protected
```

Comment:
I prefer attrs as the very fist thing, what do you think? Any other idea?

If not, it can be left this way.